WNPRC_EHR/resources/scripts/wnprc_ehr/wnprc_triggers.js

```diff
@@ -798,15 +805,26 @@ exports.init = function (EHR) {
                     }
 
                     if (row.objectid) {
-                        var msg = helper.getJavaHelper().verifyBloodVolume(row.id, row.date, draws, weights, row.objectid || null, row.quantity);
-                        if (msg != null) {
-                            if (msg.toLowerCase().indexOf('unknown weight') > -1) {
-                                volumeErrorSeverity = helper.getErrorSeverityForBloodDrawsWithoutWeight();
-                            }
+                        try {
+                            var msg = helper.getJavaHelper().verifyBloodVolume(row.id, row.date, draws, weights, row.objectid || null, row.quantity);
+                            if (msg != null) {
+                                if (msg.toLowerCase().indexOf('unknown weight') > -1) {
+                                    volumeErrorSeverity = helper.getErrorSeverityForBloodDrawsWithoutWeight();
+                                }
 
-                            //TODO: change all future bloods draws to review required, if submitted for medical purpose.
-                            EHR.Server.Utils.addError(scriptErrors, 'num_tubes', msg, volumeErrorSeverity);
-                            EHR.Server.Utils.addError(scriptErrors, 'quantity', msg, volumeErrorSeverity);
+                                //Set the severity level to INFO for limit notices strictly
+                                if (msg.indexOf('Limit') === 0 && msg.indexOf('exceeds') === -1 ) {
```

Comment:
This is picky but you could combine with the if statement on line 811 with an if / else if so that the helper function doesn't run for limit notices.